typings/index.d.ts

```diff
@@ -232,3 +283,11 @@ export default class DateTimePicker extends React.Component<
   ReactNativeModalDateTimePickerProps,
   any
 > {}
+
+export class CancelButton extends React.Component<CancelButtonPropTypes> {}
```

Comment:
What are these being exported for?

Reply:
Are you talking about the `CancelButton` and `ConfirmButton` components?

If so, well... if you try to use them in a TypeScript project, it will say it refers to a type, not a component (when passing it to `customCancelButtonIOS`or `customConfirmButtonIOS`).

Also, if you try to import the `CancelButton` or `ConfirmButton` the editor will say these components are not being exported from the declaration file.

Reply:
Gotcha! I'm not a TS expert, but given that we're already defining these types [here](https://github.com/mmazzarolo/react-native-modal-datetime-picker/blob/064ccdfc9466d79fa225572d69612c7f0e106a30/typings/index.d.ts#L15-L25), I'd update the type to be `export const` instead, instead of exporting them as classes (especially because they're not classes).

Reply:
Yeah, I did see that `CancelButtonComponent` and `ConfirmButtonComponent` were being exported, I just don't find (with all due respect) those props enough.

That's right, they're not classes. I just exported them as classes to follow the pattern in use (exporting the main component as a class), but I did not realize that the [DateTimePickerModal](https://github.com/mmazzarolo/react-native-modal-datetime-picker/blob/master/src/DateTimePickerModal.ios.js#L25) was a class. I'm sorry for that, I'm not a TS expert either.

But you're right! I'm doing this right now!